src/sys/redox/selector.rs

```diff
@@ -0,0 +1,142 @@
+use {io, Ready, PollOpt, Token};
```

Comment:
Here is where the Redox specific magic happens. We use `fevent` instead of something like `epoll` to handle events.